hw_pandas/pandas_script.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt  
+### Task 1
+# functions to read files
+
+def read_gff(path):
+    liste = ["chromosome","source","type","start","end","score","strand","phase","attributes"]
+    df_gff = pd.read_csv(path, comment='"',sep='\t',header=None,names=liste, skiprows=1)
```

Comment:
Пробелы куда-то убежали(

А почему коммент - `"`, здесь по идее должен быть `#` и skiprows не нужен, так как есть comment
```suggestion
    df_gff = pd.read_csv(path, comment='#', sep='\t', names=liste)
```